src/documents/InclusiveLearning.html.md.handlebars

```diff
@@ -39,7 +39,7 @@ Learning needs that affect learning can include:
 <figure>
 [![Floe concept design mindmap]({{{getRelativeUrl '/images/thumbs/400px-Floe_concept_design_mindmap.png' document.url}}})]({{{getRelativeUrl '/images/Floe_concept_design_mindmap.png' document.url}}})
 <figcaption>
-*Figure:* Floe concept design mindmap
+*Figure:* Floe concept design mindmap illustrates the many different ideas and parts which work together to enable a one-size-fits-one education.
```

Comment:
I think in print they also number the figures so that they can be referred to in the text. What do you think about doing that here. So instead of just "Figure:" it'd be "Figure 1:" and etc.